tracer/html.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/alecthomas/chroma/styles"
 )
 
-func (t *Tracer) GenerateHTML(dir string, res *Result) error {
+func (tr *Tracer) GenerateHTML(dir string, res *Result) error {
```

Comment:
Why change the var name? It definitely adds a lot of changed lines to the review (granted I haven't come back and cleaned up code myself)

Reply:
It was mixed t and tr across several methods. I just cleaned up all pointers to the same name for consistency.

Reply:
Ah, I only used `tr` in one place where `t` had a better meaning as a local variable. Personally, I'd rather change that than make all receivers two-char var names, but it doesn't matter much.

Reply:
I'd actually prefer `t` for the struct pointer myself. Changed accordingly